clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1661,4 +1661,36 @@ def AwaitOp : CIR_Op<"await",
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// Variadic Operations
+//===----------------------------------------------------------------------===//
+
+def VAStartOp : CIR_Op<"vastart">, Arguments<(ins CIR_PointerType:$arg_list)> {
+  let summary = "Starts a variable argument list";
+  let assemblyFormat = "$arg_list attr-dict `:` type(operands)";
+  let hasVerifier = 0;
+}
+
+def VAEndOp : CIR_Op<"vaend">, Arguments<(ins CIR_PointerType:$arg_list)> {
```

Comment:
Sorry I forgot to mention this, please change the name to `vaend` -> `va.end`, same for all other ones.